doc/migrations/3.5.0_3.5.0plus.md

```diff
@@ -83,3 +83,8 @@ For more information, please check the specs for types and functions in the afor
 * The default room config is still the same, i.e. `roomname` (default: `"Untitled"`) and `subject` (empty string).
 * The room config representation in databases (both Mnesia and RDBMS) is the same; no need for migration.
 
+## `mod_push` module is no longer available
```

Comment:
PR #2549 was just merged. In this PR new section related to `push notifications` was created. Do you think it'd be a good idea to move this section under the broader `push notifications`?